mux/crates/mux-core/src/lib.rs

```
@@ -0,0 +1,420 @@
+        self.emit(MuxEvent::TreeChanged);
+        Ok(pane)
+    }
+
+    /// Create a tab in a workspace (default: the active one).
+    pub fn new_tab(
+        self: &Arc<Self>,
+        workspace: Option<WorkspaceId>,
+        cwd: Option<String>,
+    ) -> anyhow::Result<Arc<Pane>> {
+        let pane = self.spawn_pane(cwd)?;
+        let tab_id = self.next_id();
```

Clippy (1.96.0)

[warning] unsafe function's docs are missing a `# Safety` section

(warning)